packages/ember-routing-views/lib/views/link.js

```diff
@@ -495,7 +499,7 @@ var LinkView = EmberComponent.extend({
   **/
   loadedParams: computed('resolvedParams', function computeLinkViewRouteArgs() {
     var router = get(this, 'router');
-    if (!router) { return; }
+    if (!router || !router.router) { return; }
```

Comment:
This is actually the *only* change we need to squash the issue and allow integration testing components with `{{link-to}}`s. The root problem is in the [`assert` below](https://github.com/emberjs/ember.js/pull/11829/files#diff-f3c536e535ba1d44c39353a4bc2d6496L507). However, this leaves the `{{link-to}}` in the `loading` state, which differs from the 1.13 fix.